### PR TITLE
Refactor origin configuration files.

### PR DESCRIPTION
**Pull request by bbockelm**
Give the origin config the same treatment as the cache config.  Where possible, reuse the same config files for cache and origin servers.

Didn't have a chance to test this; was hoping @matyasselmeci could take it from here.

### CODE REVIEW

File: configs/xrootd-stashcache-origin-server.cfg [hunk 1]
```diff
@@ -1,28 +1,47 @@
...
 all.role server
-all.manager redirector.osgstorage.org+ 1213
 
-xrootd.trace emsg login stall redirect
-ofs.trace all
-xrd.trace all
-cms.trace all
+
+# Drop your site-specific configuration additions into this directory.
+continue /etc/xrootd/config.d
```
Comment (jthiltges) [inline configs/xrootd-stashcache-origin-server.cfg]: In testing this PR, the daemon fails to start:
```181212 20:41:40 5603 XrootdStream: Variable originexport is undefined.
...
181212 20:41:40 5603 XrdProtocol: Protocol xrootd could not be loaded```

Looks like the `continue /etc/xrootd/config.d` needs to come before the `oss.localroot $(origindir)` and `all.export $(originexport)` entries so the variables are defined. How about moving those two blocks after `continue`?

Comment (bbockelm) [reply configs/xrootd-stashcache-origin-server.cfg]: Unfortunately, `continue` is not `include`.  The `continue` statement stops further processing of the current configuration file.

So, we may need to move these directives to the `config.d` directory.

Comment (jthiltges) [reply configs/xrootd-stashcache-origin-server.cfg]: If we set the `oss.localroot` and `all.export` parameters in `config.d/10-origin-site-local.cfg`, the service seems to run fine. I've not come up with a cleaner way yet.

Keep them as variables, so the service fails to start until after configuration?
```
# set origindir = /stash
oss.localroot $(origindir)

# set originexport = /VO
all.export $(originexport)
```

**Comment by jthiltges** [comment]
LGTM

**Pull request merged**